grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -283,6 +287,95 @@ class StockMovementController {
         render(template: "documents", model: [stockMovement: stockMovement])
     }
 
+    def comments = {
+        def stockMovement = getStockMovement(params.id)
+        render(template: "comments", model: [stockMovement: stockMovement])
+    }
+
+    def addComment = {
+        def stockMovement = getStockMovement(params.id)
+        List<User> recipients = User.list()
```

Comment:
do we return all users on purpose?

Reply:
As an example, I was following the comments on Purchase Orders which gives you a list of Users as comment receivers.

Reply:
The best case would be to use async select for this one, or I think at least you could use the `selectUser` tag (which takes only active users, I feel like there is no point in allowing selecting inactive users)